broadcast.php

```diff
@@ -15,11 +15,11 @@
 include 'header.php';
 
 // URL parameters
-$action = $_REQUEST['action'];
-$text = trim($_REQUEST['text']);
-$request_response = trim($_REQUEST['response']);
-$confirmed = $_REQUEST['confirm'];
-$communications_id = $_REQUEST['id'];
+$action = isset($_REQUEST['action']) ? $_REQUEST['action'] : '';
```

Comment:
I'm assuming these changes are more of the "proper way" of doing things in php, I dont know much about php so I trust you but do you mind explaining how/why it's better to add these checks?